src/pytest_mock_resources/cli.py

```diff
@@ -1,8 +1,14 @@
 import argparse
 import enum
+import os
 import subprocess  # nosec
 
 
+postgres_image = os.getenv("POSTGRES_IMAGE", "postgres:9.6.10-alpine")
```

Comment:
PMR proper uses https://github.com/schireson/pytest-mock-resources/blob/master/src/pytest_mock_resources/config.py#L23, which you could import here directly, it'd be PMR_POSTGRES_IMAGE.
```suggestion
postgres_image = get_env_config('postgres', 'image') or "postgres:9.6.10-alpine"
```

Reply:
@DanCardin Done.